podium-gateway/src/main/java/nl/thehyve/podium/web/rest/RequestResource.java

```diff
@@ -167,60 +167,107 @@
     }
 
     /**
-     * GET  /requests : get all the requests.
+     * GET  /requests/requester : get all the requests for which the current user is the requester.
      *
      * @param pageable the pagination information
      * @throws URISyntaxException if there is an error to generate the pagination HTTP headers
      * @return the ResponseEntity with status 200 (OK) and the list of requests in body
      */
-    @GetMapping("/requests")
+    @GetMapping("/requests/requester")
     @Timed
-    public ResponseEntity<List<RequestRepresentation>> getAllRequests(@ApiParam Pageable pageable)
+    public ResponseEntity<List<RequestRepresentation>> getRequesterRequests(@ApiParam Pageable pageable)
         throws URISyntaxException {
         UserAuthenticationToken user = SecurityUtils.getCurrentUser();
-        log.debug("REST request to get a page of Requests for user {}", user.getName());
+        log.debug("REST request to get a page of Requests for requester {}", user.getName());
         Page<RequestRepresentation> page = requestService.findAllForRequester(user, pageable);
-        HttpHeaders headers = PaginationUtil.generatePaginationHttpHeaders(page, "/api/requests");
+        HttpHeaders headers = PaginationUtil.generatePaginationHttpHeaders(page, "/api/requests/requester");
         return new ResponseEntity<>(page.getContent(), headers, HttpStatus.OK);
     }
 
     /**
-     * Fetch the request
+     * GET  /requests/status/:status/requester : get all the requests for a requester with the status.
      *
-     * FIXME: UNSECURED - Add new annotation that check whether an member (coordinator or reviewer) of an organisation
```

Comment:
@gijskant Wouldnt we want to secure this using the annotations as well? Even before calling the route. Maybe leave in the FIXME